src/components/cluster-deployments/ClusterDeploymentsListPage.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  useK8sWatchResource,
+  ListPageHeader,
+  ListPageBody,
+  VirtualizedTable,
+  TableColumn,
+  RowProps,
+  TableRow,
+  TableData,
+} from '@openshift-console/dynamic-plugin-sdk/api';
+import { Link } from 'react-router-dom';
+import { ClusterDeploymentKind } from '../../kind';
+
+const columns: TableColumn<K8sResourceCommon>[] = [
+  {
+    title: 'Name',
+  },
+  {
+    title: 'Status',
+  },
+  {
+    title: 'Distribution version',
+  },
+  {
+    title: 'Infrastructure provider',
+  },
+  {
+    title: 'Labels',
+  },
+];
+
+const ClusterDeploymentRow: React.FC<RowProps<K8sResourceCommon>> = ({
+  obj: clusterDeployment,
+  index,
+  style,
+}) => {
+  const {
+    metadata: { uid, name, namespace },
+  } = clusterDeployment;
+  return (
+    <TableRow id={uid} index={index} trKey={uid} style={style}>
+      <TableData>
+        <Link to={`/k8s/ns/${namespace}/${ClusterDeploymentKind}/${name}`}>{name}</Link>
+      </TableData>
+      <TableData>?</TableData>
```

Comment:
Let's use either dash or add at least one available and useful column here

Reply:
Done